packages/dashpay-contract/schema/dashpay.schema.json

```diff
@@ -3,6 +3,7 @@
     "type": "object",
     "indices": [
       {
+        "name": "ownerIdIndex",
```

Comment:
Let's remove "index" prefix. The size of the name matter. We probably also need to set `maxLength` to 32 or 64?